### PR TITLE
🆙 Update Twilio deps

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: do-freeze-python-deps.sh [hunk 1]
```diff
@@ -28,6 +28,6 @@ CUSTOM_COMPILE_COMMAND="./do-freeze-python-deps.sh" pip-compile --annotate --reb
 
 for rolling_package in $ROLLING_VERSION_PACKAGES; do
 
-    sed -i "s/$rolling_package==.*/$rolling_package/" $PIP_FREEZE_FILE
+    sed -i -e "s/$rolling_package==.*/$rolling_package/" $PIP_FREEZE_FILE
```
Comment (norkans7) [inline do-freeze-python-deps.sh]: https://stackoverflow.com/questions/2320564/i-need-my-sed-i-command-for-in-place-editing-to-work-with-both-gnu-sed-and-bsd

File: do-freeze-python-deps.sh [hunk 2]
```diff
@@ -28,6 +28,6 @@ CUSTOM_COMPILE_COMMAND="./do-freeze-python-deps.sh" pip-compile --annotate --reb
 
 for rolling_package in $ROLLING_VERSION_PACKAGES; do
 
-    sed -i "s/$rolling_package==.*/$rolling_package/" $PIP_FREEZE_FILE
+    sed -i '' -e "s/$rolling_package==.*/$rolling_package/" $PIP_FREEZE_FILE
```
Comment (rowanseymour) [inline do-freeze-python-deps.sh]: does this still work with the GNU sed? FYI you can switch to that sed on MacOS using `brew install gnu-sed --with-default-names`

Comment (dodobas) [reply do-freeze-python-deps.sh]: can you provide more details, why is this change required ?

Comment (rowanseymour) [reply do-freeze-python-deps.sh]: MacOS has its own `sed`.. god only knows why. You can replace it tho with Homebrew.

Comment (norkans7) [reply do-freeze-python-deps.sh]: According to https://stackoverflow.com/questions/2320564/i-need-my-sed-i-command-for-in-place-editing-to-work-with-both-gnu-sed-and-bsd this should work for both, however installing gnu-sed on OSX is a better solution

Comment (norkans7) [reply do-freeze-python-deps.sh]: @dodobas the rolling packages sed script was failing with error like `sed: 1: "pip-freeze.txt": extra characters at the end of p command`

Comment (dodobas) [reply do-freeze-python-deps.sh]: it does not ... and looks like we need a different approach to make it crossplatform ... https://stackoverflow.com/questions/2320564/i-need-my-sed-i-command-for-in-place-editing-to-work-with-both-gnu-sed-and-bsd/38595160#38595160

Comment (norkans7) [reply do-freeze-python-deps.sh]: We need the one mentioned in https://stackoverflow.com/a/2420579

**Pull request merged**